src/Administration/Resources/app/administration/src/module/sw-settings/page/sw-settings-index/index.js

```diff
@@ -13,8 +13,26 @@ export default {
     inject: [
         'acl',
         'feature',
+        'userConfigService',
     ],
 
+    data() {
+        return {
+            /**
+             * @deprecated tag:v6.8.0 - Will be removed without replacement
+             */
+            isUnitBannerHidden: false,
+            /**
+             * @deprecated tag:v6.8.0 - Will be removed without replacement
+             */
+            isLoading: true,
```

Comment:
isLoading is unnecessary, just have the `isUnitBannerHidden` as true as default (should consider rename the variable), only render the banner when the value became false

Reply:
I would go with this kind of name: `hideSettingRenameBanner: false`